src/main/scala/com/tuplejump/kafka/connect/cassandra/CassandraSinkTask.scala

```diff
@@ -51,33 +55,107 @@ class CassandraSinkTask extends SinkTask with TaskLifecycle {
 
 /** INTERNAL API. */
 private[kafka] object CassandraSinkTask {
+
   import Configuration._
 
+  import scala.util.parsing.json._
+
   /* TODO: Use keySchema, partition and kafkaOffset
    TODO: Add which types are currently supported in README */
-  def convert(record: SinkRecord, sink: SinkConfig): Query = {
+  def convert(record: SinkRecord, sink: SinkConfig, fieldMappingProperty: Option[String] = None): Query = {
+    val colNamesVsValues: Map[String, String] = {
+      fieldMappingProperty match {
+        case Some(fieldMappingString) => convertToCqlData(record, fieldMappingString)
+        case None => convertToCqlData(record)
+      }
+    }
+    colNamesVsValues.view.map(e => Vector(e._1, e._2)).transpose match {
+      case columnNames :: columnValues :: Nil =>
+        s"INSERT INTO ${sink.namespace}(${columnNames.mkString(",")}) VALUES(${columnValues.mkString(",")})"
```

Comment:
`columnValues.mkString(",")` will only work for numeric type values. In CQL, strings and timestamp type values need to be in single-quotes.